ethereum-consensus/src/bin/ec/blobs/bundler.rs

```diff
@@ -41,7 +41,7 @@ pub fn bundle(blobs: Vec<Blob>, kzg_settings: &spec::KzgSettings) -> Result<Blob
 
 // Assumes a serde_json-encoded array of `Vec<Blob>` on `reader` and uses the mainnet trusted setup.
 pub fn from_reader(reader: impl Read) -> Result<BlobsBundle, Error> {
-    let kzg_settings = spec::kzg_settings_from_json(TRUSTED_SETUP_JSON)?;
+    let kzg_settings = c_kzg::ethereum_kzg_settings(PRECOMPUTE);
```

Comment:
i'd like to encapsulate `c_kzg` inside the `crypto` module in this library

to that end, let's keep a helper function like `kzg_settings_from_json` which instead can be something like `kzg_settings_with_precompute(precompute: u64)`

this is nice as we will want to use a different `precompute` value for fulu and later forks, so each of those versions of the spec can customize via this `kzg_settings_with_precompute` function

to handle the clone issue, either have just this function, and wrap in an Arc (like it was in `ethereum-consensus/src/state_transition/context.rs`), or make a `kzg_setings_with_precompute_arc` function

i think i have a slight preference for just the one constructor